Add Moesif integration docs (#993)
wso2/docs-choreo-dev: en/docs/monitoring-and-insights/integrate-choreo-with-moesif.md

> # Integrate Choreo with Moesif

Reviewer: Since this page is under "Monitoring and Insights" just wondering whether it will be better to have a title similar to "Monitor API Insights via Moesif"

Author: Moesif not only gives API analytic features, but also monetization. I thought that someone who knows about Moesif will know what this is about.

Reviewer: Ack.

Moesif is an API analytics and monetization service that helps you grow your API products. You can integrate Choreo with Moesif and gain valuable insights into your API's usage.  While Moesif's API analytics capabilities empower you to make informed decision-making on behalf of your organization, its monetization options simplify revenue generation from your APIs, eliminating the need for complex coding. 

In this guide, you will:

- Generate a key in Moesif.
- Configure Choreo to connect to Moesif.
- Invoke an API and observe the insights on the Moesif dashboard. 

## Step 1: Generate an API key in Moesif

Let's generate an API key in Moesif. 
If you're an existing Moesif user with a registered organization, proceed to [step 1.2](#step-12-access-the-api-key-in-moesif-as-an-existing-user).

### Step 1.1: Generate a key in Moesif as a new user 

If you are a new user, follow the steps below to register an organization and generate a key:

1. Go to [https://www.moesif.com/](https://www.moesif.com/).
2. Click **Sign Up** and select an authentication option and proceed. 

> 3. In the **Get Started** page, add meaningful values to the fields and create a new organization. 

Reviewer: ```suggestion
3. On the **Get Started** page, enter meaningful values for the fields and create a new organization. 
```

    | Field                 |  Description                                            |
    |-----------------------|---------------------------------------------------------|
    | **Organization Name** |  The name of your company or team.                      |

>     | **Application Name**  | An application represents one project or environment.   |

Reviewer: Since the field is **Application Name** should we rephrase the description to perhaps say "An application name to represent your project or environment."

Author: How about, "The name of your application that represents one project or environment" ?

 4. Click **Next**.
 5. On the **Overview** page, click **Next**.

>      This opens the **Quick Install** page. As there is no data, we cannot proceed. Let's configure Moesif to receive data from Choreo. 

Reviewer: Shall we rephrase to avoid "we"

Author: Ack

 6. Select a **WSO2 Choreo** as the **Server Integration**
 7. Copy the **Application Id**. 
    
     ![First Time user API Key](../assets/img/monitoring-and-insights/first_time_key_moesif.png)
     

### Step 1.2: Access the API key in Moesif as an existing user

> If you are an existing user with a registered organization, or you do not have API Key for the organization you created,  follow the steps below to access the generated API key:

Reviewer: ```suggestion
If you are an existing user with a registered organization, or you do not have an API key for the organization you created,  follow the steps below to access the generated API key:
```
